R/c.network.list.R

```diff
@@ -0,0 +1,61 @@
+#' @rdname network.list
+#' 
+#' @param check_attr Logical: should the attributes of the combined network
+#'   lists be checked for consistency. If `TRUE` inconsistencies result in
+#'   errors.
+#' 
+#' @importFrom purrr map
+#' @export
+
+c.network.list <- function(..., check_attr = TRUE) {
+  dots <- list(...)
+  
+  # Check a list elements using all.equal or identical
+  check_list <- function(lst, fun = all.equal) {
+    r <- vapply(lst[-1], function(x) isTRUE(fun(x, lst[[1]])), logical(1))
+    all(r)
+  }
```

Comment:
This function is similar to `statnet.common::all_identical()`. Would you mind contributing it (or a version of it) there? Also, it might be worth benchmarking, because the `all_identical()`'s `for` loop seems to perform about 4 times faster.